src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/SafeMsQuicConfigurationHandle.cs

```diff
@@ -259,5 +255,38 @@ private static unsafe SafeMsQuicConfigurationHandle Create(QuicOptions options,
 
             return configurationHandle;
         }
+
+        private static QUIC_ALLOWED_CIPHER_SUITE_FLAGS CipherSuitePolicyToFlags(CipherSuitesPolicy cipherSuitesPolicy)
+        {
+            QUIC_ALLOWED_CIPHER_SUITE_FLAGS flags = QUIC_ALLOWED_CIPHER_SUITE_FLAGS.NONE;
+
+            foreach (TlsCipherSuite cipher in cipherSuitesPolicy.AllowedCipherSuites)
+            {
+                switch (cipher)
+                {
+                    case TlsCipherSuite.TLS_AES_128_GCM_SHA256:
+                        flags |= QUIC_ALLOWED_CIPHER_SUITE_FLAGS.AES_128_GCM_SHA256;
+                        break;
+                    case TlsCipherSuite.TLS_AES_256_GCM_SHA384:
+                        flags |= QUIC_ALLOWED_CIPHER_SUITE_FLAGS.AES_256_GCM_SHA384;
+                        break;
+                    case TlsCipherSuite.TLS_CHACHA20_POLY1305_SHA256:
+                        flags |= QUIC_ALLOWED_CIPHER_SUITE_FLAGS.CHACHA20_POLY1305_SHA256;
+                        break;
+                    case TlsCipherSuite.TLS_AES_128_CCM_SHA256:
+                        // not supported by MsQuic, but QUIC RFC allows it so we ignore it.
+                        break;
+                    default:
+                        throw new ArgumentException(SR.Format(SR.net_quic_unsupported_cipher_suite, cipher));
```

Comment:
What do we for SslStream if somebody sets cipher that is not allowed or supported by given system? 
My preference would probably be ignore extras instead of throwing.

Reply:
From the looks of `CipherSuitesPolicyPal.Linux` it seems that we ignore them.